File: sample/sample-native/kotlinIOS/screen/model/ApiTestScreen.swift
//
//  ApiTestScreen.swift
//  kotlinIOS
//
//  Created by hyun kim on 30/12/20.
//  Copyright © 2020 hyun kim. All rights reserved.
//

import Foundation
import SwiftUI
import sample_base

func ApiTestScreen(_ model: ApiTestViewModel) -> some View {
    Screen(model) {
        Column {
            Text("Success")
        }
    }
}

File: sample/sample-android/src/main/java/com/example/sampleandroid/view/model/ApiTestScreen.kt
package com.example.sampleandroid.view.model

import androidx.compose.foundation.layout.Column
import androidx.compose.material.Text
import androidx.compose.runtime.Composable
import kim.jeonghyeon.androidlibrary.compose.Screen
import kim.jeonghyeon.sample.viewmodel.ApiTestViewModel

@Composable
fun ApiTestScreen(model: ApiTestViewModel) {
    Screen(model) {
        Column {
            Text("Success")
        }
    }
}